### DISCUSSION TITLE
React aria components are not compatile to multi select combobox state

### DISCUSSION THREAD

**Discussion by RajaTheKing826** [adobe/react-spectrum#6994] [Q&A]
Hi all, 
I have multi select combobox requirement, as we don't have multi select combobox support in react spectrum I have written my own hooks for multi select combobox state and developed a multi select combobox.

But when i would like to convert my multi select combobox from spectrum to aria components , aria components are not accepting the multi select state and throwing the error. 

Can you give me a state which will handle multi selection and support in components according to it.

https://codesandbox.io/p/sandbox/tw478l?file=%2Fsrc%2FCombobox%2FuseMultiSelectComboBox.ts%3A36%2C31-36%2C57

**Comment by snowystinger** [comment]
This is not something we support at this time. See https://github.com/adobe/react-spectrum/issues/2140
There are suggestions for alternative accessible implementations.
